dae-alright/src/components/search-bar/search-bar.component.jsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import './search-bar.style.scss';
+import SearchDropDown from './component/dropdown/search.dropdown.component';
+import ButtonSearch from './component/button/button.component';
+import InputSearch from './component/input/input.component';
+
+class SearchBar extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.handleClick = this.handleClick.bind(this);
+        // this.handleChangeSearch = this.handleChangeSearch.bind(this);
+        // this.handleChangeFilter = this.handleChangeFilter.bind(this);       
+
+        this.state = {
+            cities: [{
+                id: '1',
+                place: 'Nigeria'}, 
+                {
+                id: '2',
+                place: 'Dubai'}, 
+                {
+                id: '3',
+                place: 'USA'}, 
+                {
+                id: '4',
+                place: 'South Africa'}],
+
+            locations: [{
+                id: '1',
+                place: 'lagos'},
+                {
+                id: '2',
+                place: 'abuja'}, 
+                {
+                id: '3',
+                place: 'ibadan'}, 
+                {
+                id: '4',
+                place: 'ondo'}, 
+                {
+                id: '5',
+                place: 'akure'}],
+
+            isFindFood: false,
+
+            searchFood: "",
+            
+            sortList: [ 
+                {
+                id: '1',
+                place: 'Name'},
+                {
+                id: '2',
+                place: 'location'},
+                {
+                id: '3',
+                place: 'cuisine'}
+
+            ]
+
+        }
+    }
+
+   handleClick() {
+       this.setState({
+           isFindFood: true
+       })
+   }
+
+    // handleChangeSearch(props) {
```

Comment:
Remove this commented code if not needed

Reply:
@samynaj ooooooo